src/hotspot/share/prims/jvmtiEnv.cpp

```diff
@@ -949,7 +949,7 @@ JvmtiEnv::SuspendThread(JavaThread* java_thread) {
   }
   if (!JvmtiSuspendControl::suspend(java_thread)) {
     // Either the thread is already suspended or
-    // the thread was in the process of exiting:
+    // it was in process of exiting.
```

Comment:
Nit: the previous comment was perfectly correct. It is okay to replace the second "thread" with "it" but it should still read "in _the_ process of exiting".

Reply:
Fixed